AffiliatedPackages.md

```diff
@@ -0,0 +1,111 @@
+# PlanetaryPy Affiliated Packages
+
+PlanetaryPy affiliated packages are an important aspect of the 
+PlanetaryPy community.
+
+An affiliated package is a planetary science related Python package that
+is not part of the `planetarypy` core package, and is not managed by the
+project but is a part of the PlanetaryPy Project community. These
+packages demonstrate a commitment to PlanetaryPy’s goals of improving
+reuse, interoperability, and interface standards for Python planetary
+science packages. In many (but not all) cases, affiliated
+packages also follow similar devlopment processes and package
+templates as for the core package.
+
```

Comment:
"as applied for (to?) the core package"?

Reply:
Wow, I just deleted the 'for' and it reads better. Fix in upcoming commit.

Reply:
sorry, just saw "devlopment" typo...